### PR TITLE
Syntax lookup @genType decorator

### PR DESCRIPTION
**Pull request by kevanstannard**
Suggestion for #162 

I've kept the description here simple, leaving the full details to the GenType docs, but I appreciate any guidance on improvements.

### CODE REVIEW

File: misc_docs/syntax/decorator_gentype.mdx [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+id: "gentype-decorator"
+keywords: ["gentype", "decorator", "typescript", "flow"]
+name: "@genType"
+summary: "This is the `@genType` decorator."
+category: "decorators"
+---
+
+The `@genType` decorator may be used to export ReScript values and types to JavaScript, and import JavaScript values and types into ReScript.
```
Comment (ryyppy) [inline misc_docs/syntax/decorator_gentype.mdx:9]:
```suggestion
The `@genType` decorator may be used to export ReScript values and types to JavaScript, and import JavaScript values and types into ReScript. It allows seamless integration of compiled ReScript modules in existing TypeScript, Flow, or plain JavaScript codebases, without loosing type information across different type systems.
```

**Review by ryyppy — COMMENTED** [review]
Looking good. Proposed some additions

**Comment by kevanstannard** [comment]
Thanks for help/corrections 👍

**Comment by ryyppy** [comment]
thanks!

**Pull request merged**